### PR TITLE
Handle lowercase project ids

### PR DESCRIPTION
**Pull request by jesperes**
Fixes bug #34 (Workzone plugin does not handle lowercase project ids), and also bug #58 (bitbucket does not preserve ssh key order).

### CODE REVIEW

**Comment by jesperes** [comment]
Note to @klarna-incubator/kred-core reviewers: this has been tested internally at Klarna (see the `sync-with-upstreams` branch in `tools-bec`).

File: src/bitbucket_http.erl [hunk 1]
```diff
@@ -80,7 +80,7 @@ do_request(Method, Url, Body) ->
   Type    = "application/json",
   Request = {Url, Headers, Type, Body},
```
Comment (Taddic) [inline src/bitbucket_http.erl:81]: Maybe put `_Body` here so that warning_as_errors won't cause compilation to fail?

**Review by Taddic — APPROVED** [review]

File: src/bitbucket_http.erl (hunk 1 above)
Comment (jesperes) [reply src/bitbucket_http.erl:81]: True, but for some weird reason, `rebar.config` uses `fail_on_warning`, which I thinks is not a valid option.
```
{erl_opts             , [fail_on_warning, debug_info]}.
```

Comment (jesperes) [reply src/bitbucket_http.erl:81]: But `Body` is actually a function argument, so this is not a problem.

**Pull request merged**